src/environments/environment.prod.ts

```diff
@@ -1,6 +1,6 @@
 export const environment = {
   production: true,
-  apiUrl: 'https://taskerapi.herokuapp.com',
-  // apiUrl: 'http://localhost:8000',
+  // apiUrl: 'https://taskerapi.herokuapp.com',
+  apiUrl: 'http://localhost:8000',
```

Comment:
Do not commit this

Reply:
@Lixing-Zhang Can you resolve the conflicts on your side?